docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Attaching Handlers to Container Lifecycle Events
+---
+
+{% capture overview %}
+
+This page shows how to attach handlers to Container lifecycle events. Kubernetes supports
+the postStart event and the preStop event. Kubernetes sends the postStart event immediately
+after a Container is started, and it sends the preStop event immediately before the
+Container is terminated.
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+### Defining postStart and preStop handlers
+
+In this exercise, you create a Pod that has one Container. The Container has handlers
```

Comment:
When does the postStart fire?  Before the containers are started?  After?  At the same time?